ext/pathname/lib/pathname.rb

```diff
@@ -585,6 +585,27 @@ def mkpath
     nil
   end
 
+  # Creates a file and the full path to the file including any intermediate directories that don't yet
+  # exist.
+  #
+  # Example:
+  #
+  #   Dir.exist?("/a/b/c") # => false
+  #
+  #   p = Pathname.new("/a/b/c/d.txt")
+  #   p.file? => false
+  #   p.touch
+  #   p.file? => true
+  #
+  #   Dir.exist?("/a/b/c") # => true
+  def touch
+    require 'fileutils'
```

Comment:
:thinking: If I understand it well, "fileutils" was "promoted" to gem and Ruby vendored code can't rely on its presence now.

Reply:
my understanding is that all builtin libraries will still be present no matter what, either within Ruby or as preinstalled gem.